themes/bootstrap5/js/lightbox.js

```diff
@@ -47,19 +61,32 @@ VuFind.register('lightbox', function Lightbox() {
     return res;
   }
 
-  // Public: Present an alert
+  /**
+   * Show a simple alert message in the lightbox.
+   * @param {string} message The message to display.
+   * @param {string} [_type] The type of alert (default = 'info').
+   */
   function showAlert(message, _type) {
     var type = _type || 'info';
     _html('<div class="flash-message alert alert-' + type + '">' + message + '</div>'
         + '<button class="btn btn-default" data-bs-dismiss="modal">' + VuFind.translate('close') + '</button>');
     _bsModal.show();
   }
+
+  /**
+   * Display a flash message within the current lightbox content.
+   * @param {string} message The message to display.
+   * @param {string} [_type='info'] The type of alert.
+   */
   function flashMessage(message, _type) {
     var type = _type || 'info';
     _modalBody.find('.flash-message,.modal-loading-overlay,.loading-spinner').remove();
     _modalBody.find('h2:first-of-type')
       .after('<div class="flash-message alert alert-' + type + '">' + message + '</div>');
   }
+  /**
```

Comment:
Should we have a newline here for consistency?

```suggestion

  /**
```